app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInTracker.kt

```diff
@@ -2,13 +2,18 @@ package com.hedvig.app.feature.loggedin.ui
 
 import com.hedvig.app.ScreenTracker
 import com.hedvig.app.feature.tracking.TrackingFacade
+import com.hedvig.hanalytics.HAnalytics
 
 class LoggedInTracker(
     private val trackingFacade: TrackingFacade,
-    private val screenTracker: ScreenTracker
+    private val screenTracker: ScreenTracker,
+    private val hAnalytics: HAnalytics,
 ) {
     fun setMemberId(memberId: String) = trackingFacade.identify(memberId)
     fun tabVisited(tab: LoggedInTabs) {
+        if (tab == LoggedInTabs.HOME) {
```

Comment:
This is temporary, and the code will look a lot more like the statements below it once all events have been properly integrated, in APP-1341.